docs/source/algorithms/CalculateEfficiencyCorrection-v1.rst

```diff
@@ -0,0 +1,238 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+This algorithm calculates the detection efficiency correction, :math:`\epsilon`, defined by:
+
+.. math::
+    :label: efficiency
+
+    \epsilon &= 1-e^{-\rho T \sigma_a (\lambda)} \\
+             &= 1-e^{-\rho_{A} \sigma_a (\lambda)} \\
+             &= 1-e^{-\alpha \lambda}
+
+and output correction is given as:
+
+.. math::
+    :label: efficiency_correction
+
+    \frac{1}{\epsilon} = \frac{1}{1-e^{-\alpha \lambda}}
+
+where :math:`\rho` is the number density in :math:`atoms/\AA^3`,
+:math:`T` is a sample thickness given in :math:`cm`,
+:math:`\sigma_a(\lambda)` is the wavelength-dependent absorption cross-section given as 
+:math:`\sigma_a (\lambda) = \sigma_a (\lambda_{ref}) \left( \frac{\lambda}{\lambda_{ref}} \right)`,
+in units of :math:`barns` and with :math:`\lambda_{ref}` = 1.7982 :math:`\AA`,
```

Comment:
Usually the math fonts are not used for units. Math formatting can be used for the exponents, though.